src/sentry/static/sentry/app/views/onboarding/project/platformpicker.jsx

```diff
@@ -19,112 +26,221 @@ class PlatformPicker extends React.Component {
-                  className={classnames({
-                    selected: this.props.platform === platform.id,
-                  })}
-                  key={platform.id}
-                  onClick={e => {
-                    analytics('platformpicker.select_platform', {platform: platform.id});
-                    this.props.setPlatform(platform.id);
-                    e.preventDefault();
-                  }}
-                />
-              );
-            })}
-          </ul>
-        ) : (
-          <p>
-            {t(
-              "Not finding your platform? There's a rich ecosystem of community supported SDKs as well (including Perl, CFML, Clojure, and ActionScript).\n Try searching for Sentry clients or contacting support."
+            ))}
+          </CategoryNav>
+          <SearchBar>
+            <InlineSvg src="icon-search" />
+            <input
+              type="text"
+              ref={this.searchInput}
+              value={filter}
+              label={t('Filter Platforms')}
+              placeholder={t('Filter Platforms')}
+              onChange={e => this.setState({filter: e.target.value}, this.logSearch)}
+            />
+          </SearchBar>
+        </NavContainer>
+        <PlatformList>
+          {platformList.map(platform => (
+            <PlatformCard
+              key={platform.id}
+              platform={platform}
+              selected={this.props.platform === platform.id}
+              onClear={e => {
+                setPlatform('');
+                e.stopPropagation();
+              }}
+              onClick={e => {
+                analytics('platformpicker.select_platform', {platform: platform.id});
+                setPlatform(platform.id);
+              }}
+            />
+          ))}
+        </PlatformList>
+        {platformList.length === 0 && (
+          <EmptyMessage
+            icon="icon-project"
+            title={t("We don't have an SDK for that yet!")}
+          >
+            {tct(
+              `Not finding your platform? You can still create your project,
+              but looks like we don't have an official SDK for your platform
+              yet. However, there's a rich ecosystem of community supported
+              SDKs (including Perl, CFML, Clojure, and ActionScript). Try
+              [search:searching for Sentry clients] or contacting support.`,
+              {
+                search: (
+                  <ExternalLink href="https://github.com/search?q=-org%3Agetsentry+topic%3Asentry&type=Repositories" />
+                ),
+              }
             )}
-          </p>
+          </EmptyMessage>
         )}
-      </div>
+      </React.Fragment>
     );
   }
 }
 
+const NavContainer = styled('div')`
+  border-bottom: 1px solid ${p => p.theme.borderLight};
+  margin-bottom: ${space(2)};
+  display: grid;
+  grid-template-columns: 1fr 300px;
+  align-items: start;
+`;
+
+const SearchBar = styled('div')`
+  ${inputStyles};
+  padding: 0 8px;
```

Comment:
lot of random values in here but if it's legacy that is fine

Reply:
This is a little less legacy, I can use `space` for some of this.

Reply:
In platform picker? I can probably use `space` for some stuff, but other stuff is just inherently "random values"

Reply:
if it's inherently random that is fine! `space` isn't a requirement and should't get in the way of stuff looking right.